workflows/rnaseq-ref-index/build-index.nf

```diff
@@ -60,14 +67,18 @@ process salmon_index_full_sa{
       -d decoys.txt \
       -i ${index_base}_k${kmer}_full_sa \
       -k ${kmer} \
-      -p ${task.cpus}
+      -p ${task.cpus} \
+      --gencode
     """
 }
 
 process kallisto_index{
   container 'quay.io/biocontainers/kallisto:0.46.2--h4f7b962_1'
   publishDir "${params.ref_dir}/kallisto_index", mode: 'copy'
-  memory 32.GB
+  memory { 120.GB * task.attempt}
+  cpus 8
+  errorStrategy { task.exitStatus in 137..140 ? 'retry' : 'terminate' }
+  maxRetries 1
```

Comment:
Point of curiousity... did you ever see it retry?

Reply:
I don't see anything in the tower output that indicates it retried... But it sure did fail with the 137 error quite a few times.